sixpack.js

```diff
@@ -199,42 +196,37 @@
         } else {
             var httpModule = url.startsWith('https') ? 'https' : 'http';
             var http = eval('require')(httpModule); // using eval to skip webpack bundling and warnings
-
             var parsedUrl = new URL(url);
             var options = {
                 protocol: parsedUrl.protocol,
                 port: parsedUrl.port,
                 hostname: parsedUrl.hostname,
                 path: parsedUrl.pathname + parsedUrl.search, 
-                headers: { Cookie: cookie }
+                headers: { Cookie: cookie },
+                timeout,
```

Comment:
Because this is not compiled by babel and could lead to syntax error in older browsers
```suggestion
                timeout
```

Reply:
done